### PR TITLE
feat : Added history page

### PR DESCRIPTION
**Pull request by aiman-ansari**
- Watched Videos added to history page.
- user can delete single video or all video from history .
Files to check
src/Pages/History/History.js
src/Pages/History/History.css
src/Pages/SingleVideo/SingleVideo.js
src/Context/HistoryContext.js

### CODE REVIEW

File: src/Pages/Explore/Explore.js [hunk 1]
```diff
@@ -6,12 +6,18 @@ import './Explore.css'
...
+
 export const Explore = () =>{
     const [isActive, setIsActive] = useState('All')
     const { categories} = useCategory()
     const { videos} = useVideo()
     const { state, dispatch} = useFilter()
-   
+    const { history, addHistory} = useHistory()
+    const {show, setShow, setIconContainer, iconContainer} = useIconContainer()
+   console.log("history",history)
```
Comment (atharvabhanage02) [inline src/Pages/Explore/Explore.js:20]: Remove console.log

File: src/Pages/History/History.js [hunk 2]
```diff
@@ -1,7 +1,60 @@
...
+import {ToastContainer} from  'react-toastify'
+import 'react-toastify/dist/ReactToastify.css';
+import { useNavigate, Link} from 'react-router-dom'
+import { useAuth } from '../../Context/AuthContext';
+import { useHistory } from '../../Context/HistoryContext';
 export const History = () =>{
+    const {isLogin} = useAuth()
+    const {history, deleteHistory, deleteAllHistory} = useHistory()
+    const navigate = useNavigate()
+    console.log(history)
```
Comment (atharvabhanage02) [inline src/Pages/History/History.js:12]: Also remove console.log from here before merging

**Review by atharvabhanage02 — APPROVED** [review]
- UI looks clean and functionalities are working fine
- Liked the use of React Toastify 
- Code is well structured

Suggestion : 
Can remove all the console.logs once the checking for desired functionality is done , Else everything looks good

**Review by sanu29 — APPROVED** [review]
Great Work.. File structure Looks good. Clean code. UI is good. CSS files are maintained properly. Naming conventions are declarative. Const used everywhere which is again a good practice. Reducer maintained Properly. Great work

**Pull request merged**